pkg/rtc/wrappedreceiver.go

```diff
@@ -67,8 +67,15 @@ type DummyReceiver struct {
 	streamId         string
 	codec            webrtc.RTPCodecParameters
 	headerExtensions []webrtc.RTPHeaderExtensionParameter
-	downtrackLock    sync.Mutex
-	downtracks       map[livekit.ParticipantID]sfu.TrackSender
+
+	downtrackLock sync.Mutex
+	downtracks    map[livekit.ParticipantID]sfu.TrackSender
+
+	settingsLock          sync.Mutex
+	maxExpectedLayerValid bool
+	maxExpectedLayer      int32
+	pausedValid           bool
+	paused                bool
```

Comment:
Adding these as I think these will be need to be set up properly once the receiver is determined.